### PR TITLE
fix the icon

### PR DESCRIPTION
**Pull request by johngruen**
Somewhere part of the icon disappeared. This should fix that.

### CODE REVIEW

**Comment by johngruen** [comment]
FIxes #177

File: data/img/overlay-player-icon.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg width="24" height="18" viewBox="0 0 24 18" xmlns="http://www.w3.org/2000/svg"><title>Combined Shape</title><path d="M0 1.995C0 .893.897 0 2.005 0h19.99C23.102 0 24 .893 24 1.995v14.01C24 17.107 23.103 18 21.995 18H2.005C.898 18 0 17.107 0 16.005V1.995zm2 3.007C2 4.45 2.455 4 2.992 4h18.016c.548 0 .992.456.992 1.002v9.996c0 .553-.455 1.002-.992 1.002H2.992C2.444 16 2 15.544 2 14.998V5.002z" fill="#FFF" fill-rule="evenodd"/></svg>
\ No newline at end of file
+<svg width="172" height="130" viewBox="0 0 172 130" xmlns="http://www.w3.org/2000/svg"><title>Combined Shape</title><path d="M0 9.998C0 4.476 4.468 0 9.99 0h152.02c5.517 0 9.99 4.475 9.99 9.998v110.004c0 5.522-4.468 9.998-9.99 9.998H9.99C4.474 130 0 125.525 0 120.002V9.998zm10 18.005A4 4 0 0 1 14.007 24h143.986A4.003 4.003 0 0 1 162 28.003v87.994a4 4 0 0 1-4.007 4.003H14.007A4.003 4.003 0 0 1 10 115.997V28.003zm8 32.002A4.007 4.007 0 0 1 22 56h64c2.21 0 4 1.8 4 4.005v47.99A4.007 4.007 0 0 1 86 112H22c-2.21 0-4-1.8-4-4.005v-47.99zM42 73c0-.553.403-.808.905-.567l23.19 11.132c.5.24.502.627 0 .868l-23.19 11.132c-.5.24-.905-.013-.905-.567V73.001z" fill="#FFF" fill-rule="evenodd"/></svg>
```
Comment (pdehaan) [inline data/img/overlay-player-icon.svg]: @jgruen, does the `<title>` tag have any significance in SVG? If so, we should maybe give it a more descriptive value than "Combined Shape". If not, ¯\\\_(ツ)\_/¯.

Comment (pdehaan) [inline data/img/overlay-player-icon.svg]: Ah, https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title

> When the current SVG document fragment is rendered as SVG on visual media, title element is not rendered as part of the graphics. **However, some user agents may, for example, display the title element as a tooltip.** Alternate presentations are possible, both visual and aural, which display the title element but do not display path elements or other graphics elements. The title element generally improve accessibility of SVG documents.

**Comment by johngruen** [comment]
@pdehaan fixed

**Comment by johngruen** [comment]
also, sheesh

**Pull request merged**